src/Components/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import "./navbar.css"
+import { NavLink } from 'react-router-dom'
+import logo from "./logo.png"
+
+const Navbar = () => {
+
+    return (
+
+        <nav className="nav-bar">
+            <NavLink exact to="/"><img src={logo} alt="ghibli-logo"/></NavLink>
+            <NavLink exact to="movies">Movies</NavLink>
```

Comment:
All to's should start with / : /movies, /locations 

They don't all need to be exact. Use exact when needed. Think about it as a someone looking at your code. They'll think there was a reason it has to be exact.